ballerina/Ballerina.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+org = "ballerina"
+name = "observe"
+version = "1.0.1"
+distribution = "slbeta3"
+export = ["observe", "observe.mockextension" ]
+
+[[platform.java11.dependency]]
+path = "../native/build/libs/observe-native-1.0.1-SNAPSHOT.jar"
+groupId = "ballerina"
+artifactId = "observe"
+
+[[platform.java11.dependency]]
+groupId = "io.opentelemetry"
+artifactId = "opentelemetry-sdk-trace"
+version = "1.0.0"
```

Comment:
Don't we need to specify the path to the external library Jar file here ? 🤔

Reply:
Downloading Maven dependencies if we do not specify the path https://ballerina.io/learn/calling-java-code-from-ballerina-and-vice-versa/#packaging-java-libraries-with-ballerina-programs